libraries/rush-lib/src/api/PackageJsonEditor.ts

```diff
@@ -107,7 +111,7 @@ export class PackageJsonEditor {
     const devDependencies: { [key: string]: string } = data.devDependencies || {};
     const resolutions: { [key: string]: string } = data.resolutions || {};
 
-    const dependenciesMeta: { [key: string]: { [key: string]: boolean } } = data.dependenciesMeta || {};
+    const dependenciesMeta: { [key: string]: object } = data.dependenciesMeta || {};
```

Comment:
Can this just reference `IDependenciesMetaYaml`? Or it probably makes more sense to define that type here and point `IDependenciesMetaYaml` to that (assuming that the types truly are the same).